CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+# SPDX-FileCopyrightText: 2022 Dawid Wróbel <me@dawidwrobel.com>
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+cmake_minimum_required(VERSION 3.12)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules)
+include(CMakeDependentOption)
+include(FeatureSummary)
+
+# set-up tool features; this needs to be done early to let the vcpkg auto-download all the optional dependencies
+option(ENABLE_ICONV "Build with ICONV" ON)
+add_feature_info("iconv" ENABLE_ICONV "Enables ICONV support for encoding conversion")
+if (ENABLE_ICONV)
+    list(APPEND VCPKG_MANIFEST_FEATURES "iconv")
+endif ()
+
+option(ENABLE_OFXCONNECT "Enable ofxconnect tool" ON)
+add_feature_info("ofxconnect" ENABLE_OFXCONNECT "Enables ofxconnect utility which allows to test OFX Direct Connect")
+if (ENABLE_OFXCONNECT)
+    list(APPEND VCPKG_MANIFEST_FEATURES "ofxconnect")
+endif ()
```

Comment:
> Move this and the corresponding ICONV lines to the end

This was purposefully moved to the front of `project(libofx)`, as required by vcpkg:

https://vcpkg.io/en/docs/users/buildsystems/cmake-integration.html#settings-reference
https://vcpkg.io/en/docs/users/buildsystems/cmake-integration.html#vcpkg_manifest_features

Reply:
Can you test that from the command line? I just did with my USAA account and got "Request unsuccessful".  I see when I dump the request to stdout instead of sending it that ofxconnect sends VERSION:102, <APPID>QWIN and <APPVER>1400 and that it has no provision for a <CLIENTUID>. USAA requires VERSION:103, <APPID>QMOFX and <APPVER>2300 along with a <CLIENTUID> which it provides as part of getting credentials. If any of that's configurable in ofxconnect it doesn't show up in `ofxconnect --help`.  Maybe AMEX is more liberal?

Reply:
The `--bank-list`, `--bank-fipid`, `--bank-services`, and `--allsupport` functions return parse errors because http://moneycentral.msn.com/money/2005/mnynet/service/ols/filist.aspx?SKU=3&VER=6 redirects to https://www.msn.com/money/. ofxserver.cpp barfs because it doesn't tell curl to follow the redirect, but even if it did the page is just MSN's financial landing page.

It could be rewritten to use https://ofxhome.com, but that's [on borrowed time](http://www.ofxhome.com/ofxforum/viewtopic.php?id=49843). I don't think there's a suitable replacement.

Reply:
There has been a recent announcement, “[New Ownership of OFXHome](https://www.ofxhome.com/ofxforum/viewtopic.php?pid=109005).”